docs/dev-proc/source-code-structure.md

```diff
@@ -47,15 +39,15 @@ Dasharo Release tags in git repository use format: `<platform>_vX.Y.Z`
 
 ## New platform support
 
-Branch for new platform should be created from most recent`master` branch tag.
-If there is justified need to create support for new board at arbitrary
-non-tagged commit developer should mark this commit with `<platform>_v0.0.0`
-tag.
+Branch for new platform should be created from the most recent commit on the
```

Comment:
We should not create branches for new platforms. Unless we want to land a new platform support and submit it via PR to dasharo branch

Reply:
fixed in 1ed0a83f08334d132029bbed91dd088fea7ee020